datasets/global-reanalysis-da.data.mdx

```diff
@@ -368,7 +368,7 @@ The output variables available on VEDA include evapotranspiration (ET), gross pr
 <Block>
 <Prose>
 ## Explore the Data
-The global reanalysis is a large dataset with nearly two decades of daily output. Here we show a comparison of two dates for a single variable. We encourage users to <Link to='/datasets/global-reanalysis-da/explore'>Explore the Data></Link> to look at different dates and to compare variables.
+The global reanalysis is a large dataset with nearly two decades of daily output. Here we show a comparison of two dates for a single variable. We encourage users to <Link to='/data-catalog/global-reanalysis-da/explore'>Explore the Data></Link> to look at different dates and to compare variables.
```

Comment:
There is a weird carrot after the word `Data`, dont think its supposed to be there. Maybe if we can move that too?